README.md

```diff
@@ -5,6 +5,7 @@
 [![][gitter img]][gitter]
 
 # AET
+#### Automated Exploratory Tests
```

Comment:
AET is no longer `Automated Exploratory Tests`. If you found any occurrence of this in documentation please create PR that removes it and replaces with "AET".